pyfar/io/io.py

```diff
@@ -504,11 +885,11 @@ def _clipped_audio_subtypes():
         return
 
     collection = {}
-    signal = pyfar.Signal([-1.5, -1, -.5, 0, .5, 1, 1.5]*100, 44100)
+    signal = pyfar.Signal([-1.5, -1, -.5, 0, .5, 1, 1.5] * 100, 44100)
```

Comment:
These changes are not related to this PR, that's a couple of beers right there. 🍻

Reply:
Yes, you got me 🤦‍♀️ , but it was inviting me to do so 🙈